backend/auth/gateway/user/http/http.go

```
@@ -109,3 +109,161 @@ func (g *userGateway) CreateNewUser(ctx context.Context, userRequestDTO dto.Crea
+
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		logger.Errorf(ctx, "failed to call request: %s", err)
+		return "", serviceresponse.NewResponseFromTemplate[any](
+			serviceresponse.RES_ERR_INTERNAL_SERVER,
+			nil,
+			nil,
+			nil,
+		)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)